src/message/__tests__/messageActionSheet-test.js

```diff
@@ -117,25 +107,15 @@ describe('constructHeaderActionButtons', () => {
 
     const buttons = constructHeaderActionButtons({
       backgroundData,
-      message,
+      message: eg.streamMessage(),
       narrow,
     });
 
     expect(buttons).toContain('unmuteStream');
   });
 
   test('show mute stream option if stream is in home view', () => {
-    const message = eg.streamMessage({
-      display_recipient: 'electron issues',
-    });
-
-    const subscriptions = deepFreeze([
-      {
-        ...eg.subscription,
-        name: 'electron issues',
-        in_home_view: true,
-      },
-    ]);
+    const subscriptions = [{ ...eg.subscription, in_home_view: true }];
```

Comment:
Did you add `in_home_view: true` for readability, since it is already true in `makeSubscription()`?

Reply:
(same answer)